testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/TopologyChangeListenerBean.java

```diff
@@ -68,7 +68,7 @@ public void establishTopology(String containerName, String cacheName, long timeo
                 long now = start;
                 long endTime = start + timeout;
                 while (!expectedMembers.equals(members)) {
-                    logger.infof("%s != %s, waiting for a topology change event. Current topology id = %d", expectedMembers, members, topology.getTopologyId());
+                    logger.debugf("%s != %s, waiting for a topology change event. Current topology id = %d", expectedMembers, members, topology.getTopologyId());
```

Comment:
I think we need to keep these at INFO since they often help debugging. @pferraro would you agree?

Reply:
yes - please leave this one as is.